main-command/src/main/scala/sbt/Watched.scala

```diff
@@ -148,6 +276,112 @@ object Watched {
     }
   }
 
+  /**
+   * Provides a number of configuration options for continuous execution.
+   */
+  trait Config {
+
+    /**
+     * A logger.
+     * @return a logger
+     */
+    def logger: Logger
+
+    /**
+     * Returns true if the continuous execution should stop.
+     * @return true if the contiuous execution should stop.
+     */
+    def shouldTerminate(): Boolean
```

Comment:
"should terminate" sounds like it should eventually terminate (finish calculation).
If it's for stopping, maybe flip the boolean and call it "isAlive"?

Reply:
Good point. I'd reused the previous method name, but you're right that is unclear.

Reply:
I was going back through the PR and realized that this method ends up being removed by the end of the PR anyway. It's just a placeholder to make the git/pr history more readable.